internal/pkg/hub/client.go

```diff
@@ -0,0 +1,28 @@
+package hub
+
+import (
+	"net/http"
+	"time"
+
+	testserver "github.com/confluentinc/cli/test/test-server"
+)
+
+// Client represents a Confluent Cloud Client as defined by ccloud-sdk-go-v2
+type Client struct {
+	URL    string
+	Debug  bool
+	Client *http.Client
+}
+
+func NewClient(isTest, unsafeTrace bool) *Client {
+	url := "https://api.hub.confluent.io"
+	if isTest {
+		url = testserver.TestHubUrl.String()
+	}
+
+	return &Client{
+		URL:    url,
+		Debug:  unsafeTrace,
+		Client: &http.Client{Timeout: 5 * time.Second},
```

Comment:
Most of our HTTP clients use a retryable client... how come we're not using that here?